OJP/OJP_JourneySupport.xsd

```diff
@@ -89,7 +89,12 @@
 			</xs:element>
 			<xs:element name="TrainNumber" type="xs:normalizedString" minOccurs="0">
 				<xs:annotation>
-					<xs:documentation>Contains the TrainNumber description from NeTEx. If several TrainNumber types exist, use the commercial number.</xs:documentation>
+					<xs:documentation>Contains the TrainNumber from NeTEx (TRAIN NUMBER from Transmodel). If several TrainNumber types exist, use the commercial one.</xs:documentation>
+				</xs:annotation>
+			</xs:element>
+			<xs:element name="VehicleNumber" type="xs:normalizedString" minOccurs="0">
+				<xs:annotation>
+					<xs:documentation>Contains the VehicleNumber of the vehicle. If several number types exist, use the commercial number. In Transmodel this may be VEHICLE.Id or VEHICLE.VehicleRegistrationNumber.</xs:documentation>
```

Comment:
This is the documentation of SIRI:

Denotes the official "registration number" of the vehicle or wagon/coach. In rail transport VEHICLE NUMBER would be equal to the 12-digit UIC wagon/coach number, possibly followed by other combinations of letters, e.g. by the UIC classification of railway coaches.

Reply:
Then I add the SIRI description?

Reply:
We should avoid being too specific here ... these numbers are evolving: we had recently a fun discussion with SNCF where they described interesting new trains having some wagons with the doors, and others with the seats so they can compose a very modular train... the fun comes when you ask how do you number these wagons ! So I would advice to just refer to "UIC wagon/coach numbering" and avoid going in further details.

Reply:
Christophe: Half number way exist. Very specialised coaches possible. specially for urban trains. Creates a big mess.  Be very open.

Reply:
Christophe: TrainNumber is for trains. 
Matthias checks, what is done with TrainNumber in NeTEx for buses and the like. 
Claus: Don't mix service and not the vehicle.  TrainNumber is a service. 
Christophe: VehicleNumber is a bad name, for a service level thing.